internal/params/gas_table.go

```diff
@@ -20,11 +20,11 @@ type GasTable struct {
 	CreateBySuicide uint64
 }
 
-// Variables containing gas prices for different ethereum phases.
+// Variables containing gas prices for different harmony phases.
 var (
-	// GasTableHomestead contain the gas prices for
-	// the homestead phase.
-	GasTableHomestead = GasTable{
+	// GasTableR3 contain the gas prices for
+	// the r3 phase.
+	GasTableR3 = GasTable{
```

Comment:
We already passed R3 phase, do we still need it?

Reply:
This is for backward compatibility